packages/react-wallet/src/components/Item/index.tsx

```diff
@@ -86,3 +90,42 @@ export function ItemLink(props: Props) {
     </div>
   );
 }
+
+interface WalletItemProps extends Props {
+  ios?: string;
+  android?: string;
+  universal?: string;
+}
+
+export function WalletItem(props: WalletItemProps) {
+  return (
+    <div>
+      {!isMobile.iOS() && !isMobile.Android() && (
+        <ItemLink
+          image={props.image}
+          title={props.title}
+          small={props.small}
+          href={props.universal}
+        />
+      )}
+
+      {isMobile.iOS() && (
+        <ItemLink
+          image={props.image}
+          title={props.title}
+          small={props.small}
+          href={props.ios || props.universal}
+        />
+      )}
+
+      {isMobile.Android() && (
+        <ItemLink
+          image={props.image}
+          title={props.title}
+          small={props.small}
+          href={props.android || props.universal}
+        />
```

Comment:
Leave only one <ItemLink /> component and make href value dynamic, you can define function for this purpose.